content/api/hub/sample_spin_wordle.md

```diff
@@ -2,7 +2,7 @@ title = "Wordle by Spin"
 template = "render_hub_content_body"
 date = "2024-08-26T04:30:32.000Z"
 content-type = "text/html"
-tags = ["javascript", "typescript", "SDK",]
+tags = ["javascript", "typescript", "sdk",]
```

Comment:
Maybe remove `sdk` here?  It doesn't seem useful unless the hub item is _for an actual SDK_ - how do we see it being used?

Reply:
done